<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>筛选文件:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="path_file"/>
      </item>
      <item>
       <widget class="QPushButton" name="do_file">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>源路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="path_work"/>
      </item>
      <item>
       <widget class="QPushButton" name="do_work">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>目标路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="path_work_target"/>
      </item>
      <item>
       <widget class="QPushButton" name="do_work_target">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="copy">
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="errorShow"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
